packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -180,6 +183,16 @@ const extractContentForAllyMessage = (props: ChatMessageContentProps): string =>
       parsedContent.replaceChild(imageTextNode, child);
     });
 
+    // Inject attachment names for aria.
+    if (attachments) {
+      let attachmentList = '';
+      /* @conditional-compile-remove(attachment-download) */
+      attachmentList = attachments.map((attachment) => attachment.name).join(', ');
+      const attachmentTextNode = document.createElement('div');
+      attachmentTextNode.innerHTML = ` ${attachmentList} `;
+      parsedContent.appendChild(attachmentTextNode);
+    }
```

Comment:
Should attachments have some prefix message such as what we have in the string "attachmentCardGroupMessage": "The message has {attachmentCount} attachment"?

Because attachment is a bit different than inline images. Inline images are inline with the message content while attachment is at the end and users might not know what the file names are for.

Reply:
i am not sure, i feel like this would be more like a spec from a11y team - i.e. what is the preferred way to announce attachments, they could also argue that they want file names and not counts.